src/pyedb/dotnet/edb_core/utilities/simulation_setup.py

```diff
@@ -103,10 +110,10 @@ def _set_edb_setup_info(self, edb_setup_info):
             "kQ3D": None,
             "kNumSetupTypes": None,
         }
-        if self._pedb.edbversion:
-            version = self._pedb.edbversion.split(".")
-            if int(version[0]) == 2024 and int(version[1]) == 2 or int(version[0]) > 2024:
-                setup_type_mapping["kRaptorX"] = utility.RaptorXSimulationSetup
+
+        version = self._pedb.edbversion.split(".")
+        if int(version[0]) == 2024 and int(version[1]) == 2 or int(version[0]) > 2024:
+            setup_type_mapping["kRaptorX"] = utility.RaptorXSimulationSetup
```

Comment:
Aren't we missing the mapping to "kHFSSPI" here ?

Reply:
@gkorompi can you add kHFSSPI in another PR?